action/protocol/execution/protocol_test.go

```diff
@@ -402,6 +402,11 @@ func (sct *SmartContractTest) prepareBlockchain(
 	}
 	if sct.InitGenesis.IsLondon {
 		// London is enabled at okhotsk height
+		cfg.Genesis.Blockchain.JutlandBlockHeight = 0
+		cfg.Genesis.Blockchain.KamchatkaBlockHeight = 0
```

Comment:
this height fix the map order bug, so need to enable for properly running Berlin/London tests